test/specs/bin/cli.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const BB = require('bluebird')
+const test = require('tap').test
+const requireInject = require('require-inject')
+
+test('cli: invokes main with parsed options', t => {
+  class FakeInstaller {
+    constructor (opts) {
+      this.opts = opts
+    }
+    run () {
+      t.comment('opts:', this.opts)
+      t.is(this.opts.ignoreScripts, false, 'ignoreScripts defaults to false')
+      t.is(this.opts.offline, false, 'offline defaults to false')
+      return BB.resolve({count: 0, time: 0})
```

Comment:
You should put a `t.plan(2)` at the very top of the test(), to confirm that the assertions you added did, in fact, run. Otherwise, your test will pretend it succeeded even if `run()` itself didn't run.

Reply:
True, though that would require `cli()` to return an un-rejected promise on its own.

Either way, `t.plan()` is a good call 👍

Reply:
Done.